lib/engine/ability/base.rb

```diff
@@ -10,7 +10,7 @@ class Base
       include Ownable
 
       attr_accessor :count_this_or, :description
-      attr_reader :type, :owner_type, :remove, :when, :count, :count_per_or, :start_count
+      attr_reader :type, :owner_type, :remove, :when, :count, :count_per_or, :start_count, :show_count
```

Comment:
why do we need to add show_count? what was wrong with the original

Reply:
I want to use start_count in other places and it was dependent on show_count being true. This allows me to break that dependency without affecting the show_count logic.

Reply:
Discussed further in slack. I think we are on the same page.